packages/devextreme/js/ui/popup/ui.popup.js

```diff
@@ -491,15 +497,17 @@ const Popup = Overlay.inherit({
 
             item.toolbar = data.toolbar || item.toolbar || 'top';
 
-            if(item && item.toolbar === toolbar) {
+            if(item?.toolbar === toolbar) {
                 if(isShortcut) {
                     extend(item, { location: data.location }, this._getToolbarItemByAlias(data));
-                }
-
-                const isLTROrder = currentPlatform === 'generic';
-
-                if((data.shortcut === 'done' && isLTROrder) || (data.shortcut === 'cancel' && !isLTROrder)) {
```

Comment:
do we ignore RTL now?

Reply:
no. There is wrong name of variable. it's not a check on RTL. it's check on theme. right now we have decided what order of buttons should not depends on themes